examples/config/many_well.yaml

```diff
@@ -1,52 +1,53 @@
-# with relatively small values so I can run things locally
```

Comment:
This is the config that I'm using for the many well experiment - I overwrite specifics (e.g. loss type) in the Makefile. My GPUs are quite small - so yours may be better with a bigger batch size?